designer/client/src/list/ListEdit.tsx

```diff
@@ -200,6 +213,73 @@ export function ListEdit() {
           errorMessage={errors.title}
         />
 
+        {selectedList.items.length ? (
+          <div className="govuk-form-group">
+            <span className="govuk-label govuk-label--s">
+              {i18n('list.type')}
+            </span>
+            <p className="govuk-body">
+              {`${
+                listTypeRadioItems.find(
+                  (item) => item.value === selectedList.type
+                )?.labelText
+              } values only`}
+            </p>
+          </div>
+        ) : (
+          <div className="govuk-form-group">
+            <fieldset className="govuk-fieldset">
+              <legend className="govuk-fieldset__legend govuk-fieldset__legend--s">
+                {i18n('list.type')}
+              </legend>
+              <div id="list-type-hint" className="govuk-hint">
+                {i18n('list.typeHint')}
+              </div>
+              <div
+                id="list-type"
+                className="govuk-radios govuk-radios--small"
+                data-module="govuk-radios"
+              >
+                {listTypeRadioItems.map((item) => {
+                  const name = 'type'
+                  const id = `${name}-${item.value}`
+
+                  return (
+                    <div className="govuk-radios__item" key={item.value}>
```

Comment:
React can cache and skip re-rendering by key

Is using the value definitely unique enough?

```suggestion
                    <div className="govuk-radios__item" key={`${list.name}-${id}`}>
```